js/apps/admin-ui/src/user/UserRoleMapping.tsx

```diff
@@ -36,7 +36,7 @@ export const UserRoleMapping = ({ id, name }: UserRoleMappingProps) => {
             }),
           ),
       );
-      addAlert(t("roleMappingUpdatedSuccess"), AlertVariant.success);
+      addAlert(t("userRoleMappingUpdatedSuccess"), AlertVariant.success);
     } catch (error) {
       addError("clients:roleMappingUpdatedError", error);
```

Comment:
This label has not been updated.

Reply:
Talked to Erik, this should be handled by the fallback namespace. We'll refactor this in a future PR, as it increases the chances this PR becomes unmergable.